Block/Menu/Edit/Tab/Nodes.php

```diff
@@ -141,8 +141,13 @@ private function renderNodeList($level, $parent, $data)
         $nodes = $data[$level][$parent];
         $html = '<ul>';
         foreach ($nodes as $node) {
-            $html .= '<li class="jstree-close" data-type="' . $node->getType() . '" data-content="' . $node->getContent(
-                ) . '" data-classes="' . $node->getClasses() . '" id="node_' . $node->getId() . '"">';
+            $html .= '<li class="jstree-close"';
+            $html .= 'data-type="' . $node->getType() . '"';
+            $html .= 'data-content="' . $node->getContent() . '"';
+            $html .= 'data-classes="' . $node->getClasses() . '"';
+            $html .= 'data-target="' . $node->getTarget() . '"';
+            $html .= 'id="node_' . $node->getId() . '""';
```

Comment:
You have double quotes twice here

Reply:
Better build this using array. See https://github.com/SnowdogApps/magento2-menu/blob/master/Block/Menu.php#L285

Reply:
double quotes, @dawiddczaja can we build this using array ?

Reply:
I think this line is correct https://github.com/SnowdogApps/magento2-menu/blob/master/Block/Menu.php#L315 @DariuszMaciejewski What You think about that ?